internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -378,6 +386,13 @@ func (h *Host) modifyHostSettings(ctx context.Context, containerID string, req *
 			return errors.New("the request's settings are not of type LCOWSecurityPolicyEnforcer")
 		}
 		return h.SetSecurityPolicy(r.EnforcerType, r.EncodedSecurityPolicy)
+	case guestresource.ResourceTypePolicyFragment:
+		r, ok := req.Settings.(*ctrdtaskapi.PolicyFragment)
+		if !ok {
+			return errors.New("the request settings are not of type PolicyFragment")
+		}
+		log.G(ctx).WithField("settings", r).Debug("policy fragment received in guest")
```

Comment:
IMO we should do the logging in `InjectFragment` if needed, and keep this as boilerplate.

Reply:
oops, forgot to remove some logging statements.